crutest/src/main.rs

```diff
@@ -804,6 +806,7 @@ async fn main() -> Result<()> {
         }
         _ => slog::Level::Info,
     };
+
```

Comment:
nit: remove newline unless it's intentional, to minimize the diff

Reply:
Fixed